packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/collect_multi_namespace_references.ts

```diff
@@ -307,6 +304,9 @@ async function optionallyUseSecurity(
     // Is the user authorized to access this object in this space?
     let isAuthorizedForObject = true;
     try {
+      // ToDo: this is the only remaining call to enforceAuthorization outside of the security extension
+      // This was a bit complicated to change now, but can ultimately be removed when authz logic is
+      // migrated from the repo level to the extension level.
       securityExtension.enforceAuthorization({
```

Comment:
This is the only remaining call to enforceAuthorization outside of the security extension. We should be able to make this method private once we have migrated the authz logic from the repository-level into the extension.